newdle/schemas.py

```diff
@@ -21,11 +21,14 @@ class UserSchema(mm.Schema):
     name = fields.Function(lambda u: f'{u["first_name"]} {u["last_name"]}')
     initials = fields.Function(lambda u: f'{u["first_name"][0]} {u["last_name"][0]}')
     uid = fields.String()
+    avatar_url = fields.Function(
+        lambda user: url_for('api.user_avatar', email=user['email'])
+    )
```

Comment:
Do we still need this? We have access to both name and email on the client side anyway.

Anyway, the fact that we have it here means that the topic of authentication we discussed earlier suddenly becomes super easy: You can just include a signature in the URL and use this to avoid creating an unrestricted image proxy.